charts/kubeai/values.yaml

```diff
@@ -97,6 +98,19 @@ resourceProfiles:
         operator: "Equal"
         value: "present"
         effect: "NoSchedule"
+  nvidia-gpu-gh200:
+    imageName: "gh200"
+    limits:
+      nvidia.com/gpu: "1"
+    requests:
+      nvidia.com/gpu: "1"
+    tolerations:
+      - key: "nvidia.com/gpu"
+        operator: "Equal"
+        value: "present"
+        effect: "NoSchedule"
+    nodeSelector:
```

Comment:
could you remove this nodeSelector for the default values.yaml?  The reason is because this wouldn't work for users that aren't using the same node labels. We try to move any environment specific nodeSelectors to separate values files, see for example values-gke.yaml and values-eks.yaml. 

Let's create a values-nvidia-device-plugin.yaml that adds these kind of nodeSelectors.  It seems anyone using the values-nvidia-device-plugin could technically use these. Note that for GKE NAP and EKS karpetner this specific nodeSelector wouldn't work as expected.

Reply:
Removed the nodeselector